blazor/mention/code-snippet/minimum-filter-char.razor

```diff
@@ -0,0 +1,39 @@
+@using Syncfusion.Blazor.DropDowns
+@using Syncfusion.Blazor.Data
+
+<SfMention TItem="EmployeeData" PopupWidth="250px" SuggestionCount=6 MinLength=3>
+    <TargetComponent>
+        <div id="commentsMention" placeholder="Type @@ and tag employee"></div>
+    </TargetComponent>
+    <ChildContent>
+        <SfDataManager Url="https://ej2services.syncfusion.com/production/web-services/api/Employees" Adaptor="Syncfusion.Blazor.Adaptors.WebApiAdaptor" CrossDomain="true"></SfDataManager>
+        <MentionFieldSettings Text="FirstName" Value="EmployeeID"></MentionFieldSettings>
+    </ChildContent>
+</SfMention>
+
+<style>
+
+    #commentsMention {
+        min-height: 100px;
+        border: 1px solid #D7D7D7;
+        width: 600px;
+        font-size: 14px;
+        padding: 8px;
+        border-radius: 4px;
+    }
+
+    div#commentsMention[placeholder]:empty:before {
+        content: attr(placeholder);
+        color: #555;
+    }
+
+</style>
+
+@code {
+
+    public class EmployeeData
+    {
+        public string FirstName { get; set; }
+        public string Country { get; set; }
+    }
```

Comment:
EmployeeID - add it here. Ensure the all code snippets

Reply:
Addressed